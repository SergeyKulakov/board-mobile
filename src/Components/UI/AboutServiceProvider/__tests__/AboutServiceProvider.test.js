import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { AboutServiceProvider as Component } from '../index'

describe('AboutServiceProvider', () => {
  const defaultProps = {
    comments: 0,
    distance: {
      length: 12,
      units: 'km',
    },
    jobsDoneCount: 3,
    star: 4,
    description: undefined,
    onCommentsClick: () => null,
  }

  it('render snapshots', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    const wrapper2 = shallow(
      <Component {...defaultProps} description="description" />,
    )

    expect([wrapper, wrapper2]).toMatchSnapshot()
  })
})
