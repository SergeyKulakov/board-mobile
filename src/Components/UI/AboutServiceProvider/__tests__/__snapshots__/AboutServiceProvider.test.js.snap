// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutServiceProvider render snapshots 1`] = `
Array [
  <Styled(View)>
    <Styled(View)>
      <Styled(View)>
        <Styled(Styled(Text))>
          3
        </Styled(Styled(Text))>
        <Styled(Text)>
          Jobs Done
        </Styled(Text)>
      </Styled(View)>
      <Styled(View)>
        <Styled(View)>
          <StarsIcon />
        </Styled(View)>
        <Styled(TouchableOpacity)
          onPress={[Function]}
        >
          <CommentsIcon
            value={0}
          />
        </Styled(TouchableOpacity)>
        <Styled(View)>
          <DistanceIcon
            icon={
              Object {
                "size": 15,
              }
            }
            units="km"
            value={12}
          />
        </Styled(View)>
      </Styled(View)>
    </Styled(View)>
  </Styled(View)>,
  <Styled(View)>
    <Styled(View)>
      <Styled(View)>
        <Styled(Styled(Text))>
          3
        </Styled(Styled(Text))>
        <Styled(Text)>
          Jobs Done
        </Styled(Text)>
      </Styled(View)>
      <Styled(View)>
        <Styled(View)>
          <StarsIcon />
        </Styled(View)>
        <Styled(TouchableOpacity)
          onPress={[Function]}
        >
          <CommentsIcon
            value={0}
          />
        </Styled(TouchableOpacity)>
        <Styled(View)>
          <DistanceIcon
            icon={
              Object {
                "size": 15,
              }
            }
            units="km"
            value={12}
          />
        </Styled(View)>
      </Styled(View)>
    </Styled(View)>
    <Styled(View)>
      <MoreText>
        description
      </MoreText>
    </Styled(View)>
  </Styled(View)>,
]
`;
